CONTRIBUTING.md

```diff
@@ -28,12 +29,26 @@ npm install
 
 #### Run the dev script
 
-To initiate the development environment, you can use the following script. This script will first check if bos-loader is installed and, if not, install it. Then, it will serve the default development environment.
+To initiate the development environment, you can use the following script. This script will first check if bos-loader is installed and, if not, install it. Then, it will serve the default development environment and watch for changes inside `./src`.
 
-```sh
+```zsh
```

Comment:
nit: I believe `sh` is generic enough, so I don't see a justification for `zsh`. I don't want to have a mix of "sh" in the READMEs, so I suggest to keep them all `sh`.